api/v1alpha4/virtualmachine_types.go

```diff
@@ -883,6 +936,12 @@ type VirtualMachineStatus struct {
 
 	// Storage describes the observed state of the VirtualMachine's storage.
 	Storage *VirtualMachineStorageStatus `json:"storage,omitempty"`
+
+	// +optional
+
+	// TaskID describes the ID of a task created by VM Operator to modify this
```

Comment:
Suggest:

```
// TaskID describes the observed ID of the task created by VM Operator to
// perform some long-running operation on the VM.
```

I do not want to say "modify" because this ID might also surface the ID of a snapshot task for instance.